docs/src/main/paradox/stream/operators/Sink/forall.md

```diff
@@ -0,0 +1,47 @@
+# Sink.forall
+
+A `Sink` that will test the given predicate `p` for every received element and completes with the result.
+
+@ref[Sink operators](../index.md#sink-operators)
+
+## Signature
+
+@apidoc[Sink.forall](Sink$) { scala="#forall%5BT%5D(p%3A%20T%20%3D%3E%20Boolean):org.apache.pekko.stream.scaladsl.Sink[T,scala.concurrent.Future[Boolean]]" java="#forall(org.apache.pekko.japi.function.Predicate)" }
```

Comment:
```suggestion
@apidoc[Sink.forall](Sink$) { scala="#forall[T](p:T=%3EBoolean):org.apache.pekko.stream.scaladsl.Sink[T,scala.concurrent.Future[Boolean]]" java="#forall(org.apache.pekko.japi.function.Predicate)" }
```

Can you update this? It can be simpler, And I have been verifyed locally.

```
[info] compiling 81 Scala sources and 1 Java source to /Users/roiocam/IdeaProject/incubator-pekko/bench-jmh/target/scala-2.13/classes ...
[warn] Duplicate anchor [#bug-fixes] on [release-notes/index.html]
[success] Total time: 240 s (04:00), completed 2024-1-27 21:36:02

```